pkg/ui/workspaces/cluster-ui/src/databaseDetailsV2/index.tsx

```diff
@@ -52,9 +57,23 @@ export const DatabaseDetailsPageV2 = () => {
     },
   ];
 
+  const dbName =
+    error?.status === 404 ? "Database Not Found" : data?.metadata.db_name;
```

Comment:
If data is nullable, do you think we should change this to something like
```suggestion
    error?.status === 404 || !data ? "Database Not Found" : data.metadata.db_name;
```
I assume this case would be hit in the situation where a non 404 error status is occurs

Reply:
not blocking: Do we have any sort of "Not Found" UI states? For example, if you went to `#/database/10000` (or database that doesn't exist), it seems like it should render a "Not Found" page within the page layout.

Reply:
Looked around and we have this component it: https://github.com/cockroachdb/cockroach/blob/d5f328ea6f3efd8fbe631c97d59f7b74307d22f9/pkg/ui/workspaces/db-console/src/views/app/components/errorMessage/notFound.tsx#L12.

But it seems like it's used specifically for routes that don't match a page - not sure if we typically show it when an api response returns with 404. This is probably something we should ask design.

Applied suggestion!